crates/runner-host/crates/web-client/src/board.rs

```diff
@@ -0,0 +1,95 @@
+// Copyright 2023 Google LLC
```

Comment:
(same in all new files)
```suggestion
// Copyright 2024 Google LLC
```

Reply:
Fixed